crates/re_types/src/components/view_coordinates_ext.rs

```diff
@@ -259,6 +259,13 @@ impl std::str::FromStr for ViewCoordinates {
     }
 }
 
+impl Default for ViewCoordinates {
+    #[inline]
+    fn default() -> Self {
+        Self::RFU
```

Comment:
This is a problem with the current `ViewCoordinates`: what is a nice default for camera coordinates is not necessarily the nicest default for 3D world coordinates.

We could add a TODO linking to https://github.com/rerun-io/rerun/issues/2663 here, and explain for what usecase this is a good default (world coords, not camera coords)